resources/lang/es/activity.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+return [
+    'causer_type'  => 'Tipo de causante',
+    'causer_id'    => 'Id del causante',
+    'subject_type' => 'Tipo de objeto sometido',
+    'subject_id'   => 'Id del objeto sometido',
+    'description'  => 'Descripción',
+    'attributes'   => 'Atributos / valor(es) actual(es) o nuevo(s)',
+    'old'          => 'Valor(es) anterior(es)',
+    'subject'      => "Asunto",
```

Comment:
Overly picky but to match psr-12 standard can you please use single quotes to wrap values? Example: `'Asunto'`

Reply:
All right. I have removed the double quotes to use single quotes to wrap values.